### PR TITLE
3371 Spike Cypress testing

### PR DESCRIPTION
**Pull request by liamtoozer**
# Motivation and Context

We want to start testing our response operations UI in our CI environment, using the [tests from a previous spike](https://github.com/ONSdigital/ssdc-rm-response-operations/blob/b22145f4a59c383862a0d47cc73b4f7207ba2290/ui/cypress/integration/CreateSurvey.js)

**Note**: There are currently no TravisCI checks - this will be handled in future tickets. This PR is simply a means to get a working test up and running in our CI

# What has changed

* Copied across test from the older spike and configured to allow `baseURL` to be used
* Added Dockerfile, pod manifest, and task ready for CI

# How to test?

Checkout the other branches and follow the instructions on the ticket

# Links

[Trello](https://trello.com/c/sHrZEWg3)

### CODE REVIEW

**Review by AdamHawtin — CHANGES_REQUESTED** [review]
I think some README words on pre-requisites/running/developing the tests are needed.

**Review by AdamHawtin — COMMENTED** [review]
I've also got an intermittent failure locally with 
```     AssertionError: Timed out retrying after 4000ms: expected 'http://localhost:7777/createsurvey' to include '/surveys?flashMessageUntil'```

I can see in the test video (great feature!) that the page with the flashed message did load, but slightly after that test assertion failed, I think the test perhaps just didn't wait quite long enough for the redirect, since it depends on a pubsub confirmed publish it can take a couple of seconds to redirect to that page. Can the tests be made to allow a longer timeout on that assertion to make them less flaky?

**Review by ryangrundy7 — APPROVED** [review]
Works locally and in GCP 👍

**Review by davidmort — APPROVED** [review]
this looks good and think it will benefit us a lot when we start doing more ui stuff, we will need to think about how we want this do work in our pipelines and how we manage all the screenshot/video files that get generated, but this is a good start

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+build:
+	docker build -t europe-west2-docker.pkg.dev/ssdc-rm-ci/docker/ssdc-rm-ui-tests .
+
+install:
+	npm install cypress@9.6.0
```
Comment (AdamHawtin) [inline Makefile]: I think this should just be an `npm install`, the required package should be specified in the package.json not in this command.

**Review by AdamHawtin — APPROVED** [review]

**Pull request merged**